src/fu-util-common.h

```diff
@@ -24,6 +24,14 @@ typedef struct {
 	FuUtilCmdFunc	 callback;
 } FuUtilCmd;
 
+typedef enum {
+	FU_SECURITY_ATTR_TO_STRING_FLAG_NONE			= 0,
+	FU_SECURITY_ATTR_TO_STRING_FLAG_SHOW_OBSOLETES		= 1 << 0,
+	FU_SECURITY_ATTR_TO_STRING_FLAG_SHOW_URLS		= 1 << 1,
+	/*< private >*/
+	FU_SECURITY_ATTR_TO_STRING_FLAG_LAST
+} FuSecurityAttrToStringFlags;
+
```

Comment:
If adding all this information specific to the command line client, I'd argue that `--verbose` or something similar should add the URL too

Reply:
Maybe just tie it to `--show-all` as well?

Reply:
Sure.